charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -77,6 +77,7 @@ controller:
   # If the default is not set and fstype is unset in the StorageClass, then no fstype will be set
   defaultFsType: ext4
   env: []
+  envFrom: []
```

Comment:
Can you include a comment here for clarity? Something like "envFrom provides a way to reference configMaps across multiple resources".

Reply:
Added "Use envFrom to reference ConfigMaps and Secrets across all containers in the deployment"